infermedica/component-library
# refactor: get rid of fast-deep-equal-library, and use fast-equals instead

## Description
- gets rid of `fast-deep-equal` library,
- uses `fast-equals` library instead,
- updates some packages with no conflicts and major changes, 

## Related Issue
Closes #392 

## How Has This Been Tested?
- Run `pnpm serve`, and check if everything still works.
- Run `pnpm check:types` and see no errors.
- Clicked through the components in Storybook.

## Checklist:
